### DISCUSSION TITLE
PS/XBOX Save Files

### DISCUSSION THREAD

**Discussion by tnkr-uk** [miragedmuk/ASV#4] [General]
I have tried to run a console save file through ASV but was hit with 'Content failed to load'.

Was curious if the file could be looked into to see if ASV could support it?
I would only imagine that it would be the same if not very similar as the steam version. 

My discord name is 'tnkr.',
If you wish to shoot me a message on there that would be greatly appreciated 👍🏽.

Thanks!

**Comment by miragedmuk** [comment]
Sent a discord friend request.  If you can provide me a copy of your .ark save file I can investigate.
